app/models/media.js

```diff
@@ -49,5 +50,47 @@ export default Base.extend({
       const decimal = curr.toFixed(1).toString();
       return prev + (parseInt(freqs[decimal], 10) || 0);
     }, 0);
+  }).readOnly(),
+
+  season: computed('startDate', function() {
+    const start = get(this, 'startDate');
+    if (!start) { return null; }
+    const month = start.month() + 1;
+    if ([12, 1, 2].includes(month)) {
+      return 'winter';
+    } else if ([3, 4, 5].includes(month)) {
+      return 'spring';
+    } else if ([6, 7, 8].includes(month)) {
+      return 'summer';
+    } else if ([9, 10, 11].includes(month)) {
+      return 'fall';
+    }
+  }).readOnly(),
+
+  seasonYear: computed('season', function() {
+    const season = get(this, 'season');
+    if (!season) { return null; }
+    const start = get(this, 'startDate');
+    const year = start.year();
+    const month = start.month();
+    if (season === 'winter' && month === 12) {
+      return year + 1;
+    }
+    return year;
+  }).readOnly(),
```

Comment:
ah whoops i was initially going to calculate upcoming based on season, and forgot to move them back when i switched. will do that shortly.